libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -48,7 +47,7 @@ class AC_PosControl
 public:
 
     /// Constructor
-    AC_PosControl(const AP_AHRS_View& ahrs, const AP_InertialNav& inav,
+    AC_PosControl(AP_AHRS_NavEKF& ahrs, const AP_AHRS_View& ahrs_view,
```

Comment:
View is definitely needed as rotation is applied for quadplanes - that's why it was created. But I think that only the roll/pitch are rotated, not everything else, so we wouldn't need the rotated and not-rotated AHRS objects. We should get confirmation from @tridge.